cfs_rust_amalgamated/src/main.rs.diff

```diff
@@ -0,0 +1,45 @@
+diff --git a/cfs_rust_amalgamated/src/main.rs b/cfs_rust_amalgamated/src/main.rs
+index 464f673..d111cd0 100644
+--- a/cfs_rust_amalgamated/src/main.rs
++++ b/cfs_rust_amalgamated/src/main.rs
+@@ -17294,7 +17294,7 @@ pub unsafe extern "C" fn UT_DefaultStubImplWithArgs(
+                 LocalContext.Int32StatusCode,
+                 Counter,
+                 &mut LocalContext,
+-                ArgListCopy.as_va_list(),
++                core::mem::transmute(ArgListCopy.as_va_list()),
+             );
+         } else {
+             LocalContext.Int32StatusCode = ((*StubPtr).Data.Cb.Ptr.SimpleHook)
+@@ -17317,7 +17317,7 @@ pub unsafe extern "C" fn UT_DefaultStubImplWithArgs(
+                 (*StubPtr).Data.Cb.CallbackArg,
+                 FuncKey,
+                 &mut LocalContext,
+-                ArgList.as_va_list(),
++                core::mem::transmute(ArgList.as_va_list()),
+             );
+         } else {
+             ((*StubPtr).Data.Cb.Ptr.SimpleHandler).expect("non-null function pointer")(
+@@ -29945,7 +29945,7 @@ pub unsafe extern "C" fn CFE_ES_PoolCreateEx(
+     if Status != 0 as libc::c_int {
+         return Status;
+     }
+-    Alignment = &mut (*(0 as *mut C2RustUnnamed_25)).Align as *mut CFE_ES_PoolAlign_t as cpuaddr;
++    Alignment = core::mem::align_of::<CFE_ES_PoolAlign_t>() as _;
+     if Alignment < 4 as libc::c_int as libc::c_ulong {
```

Comment:
```suggestion
     if Alignment < 4 as libc::c_int as size_t {
```
from #2 ?

Reply:
Yeah, sure, but I can just merge in this change from #2 next, right?